developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -134,145 +134,179 @@ And here is how it should look in your DAV request:
+|                               | | Should return the list of lock, but           |                                                                          |
+|                               | | always return an empty response.              |                                                                          |
++-------------------------------+-------------------------------------------------+--------------------------------------------------------------------------+
+| <d:supportedlock />           | | Dummy endpoint for class 2 WebDAV support.    | | ``<d:lockentry>``                                                      |
+|                               | | Always provide the same lock capabilities.    | |     ``<d:lockscope><d:exclusive /></d:lockscope>``                     |
+|                               | |                                               | |     ``<d:locktype><d:write /></d:locktype></d:lockentry>``             |
+|                               | |                                               | | ``</d:lockentry>``                                                     |
+|                               | |                                               | | ``<d:lockentry>``                                                      |
+|                               | |                                               | |     ``<d:lockscope><d:shared /></d:lockscope>``                        |
+|                               | |                                               | |     ``<d:locktype><d:write /></d:locktype>``                           |
+|                               | |                                               | | ``</d:lockentry>``                                                     |
++-------------------------------+-------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:id />                     | | The fileid namespaced by the instance id.     |                                                                          |
+|                               | | Globally unique.                              |                                                                          |
++-------------------------------+-------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:fileid />                 | The unique id for the file within the instance. |                                                                          |
++-------------------------------+-------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:downloadURL />            | The URL to download the file.                   |                                                                          |
```

Comment:
These could have some example values

Reply:
No example as not implemented yet by any app. Added a note about that.